src/main/java/com/coravy/hudson/plugins/github/GithubLinkAction.java

```diff
@@ -1,12 +1,13 @@
 package com.coravy.hudson.plugins.github;
 
-import java.util.Collection;
-import java.util.Collections;
-
 import hudson.Extension;
 import hudson.model.Action;
 import hudson.model.Job;
 import jenkins.model.TransientActionFactory;
+import org.jenkinsci.plugins.github.util.XSSApi;
+
+import java.util.Collection;
+import java.util.Collections;
```

Comment:
Yes, but re-ordering mean that it not defined in checkstyle. http://checkstyle.sourceforge.net/config_imports.html#ImportOrder_Example